pkg/asset/cluster/tfvars.go

```diff
@@ -673,12 +673,17 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		}
 
 		labels := kubevirtutils.BuildLabels(clusterID.InfraID)
+		interfaceBindingMethod := "InterfaceBridge"
+		if installConfig.Config.Kubevirt.InterfaceBindingMethod != "" {
+			interfaceBindingMethod = "Interface" + installConfig.Config.Kubevirt.InterfaceBindingMethod
+		}
```

Comment:
Should this value be set here, or in the machineproviderspec (machines.go) and then read here? There is a framework for setting default values in the machine providers. machines.go. It seems like in this current code there would be a mismatch between your masters created by terraform and the machine provider spec which doesn't seem correct.

Reply:
Fixed, move that logic to `pkg/asset/machines/kubevirt/machines.go`.
Had to change the signature of `provider` function to pass the installConfig.